### ISSUE TITLE
What will be the price options

### ISSUE THREAD

**Issue by dadhi** [gmasterscm/ama#3]
Questions:

- The price options or plans in general
- Options for OSS developers

**Comment by psantosl** [comment]
We are still deciding on pricing.

What is clear is that we'll have a free version for OSS projects, and that the general price for commercial use will be super affordable.

I'll share more info as soon as we have it :)

**Comment by Devonthemoon** [comment]
Good question!
* we are working on the pricing plans, and we'll release them in the coming weeks. 
* OSS developers working on OSS projects will use gmaster free 
Stay tuned!
Thanks

**Comment by dadhi** [comment]
One additional question, 

What will be the deal (if any) for SemanticMerge standalone tool for GMaster users?

**Comment by Devonthemoon** [comment]
Thanks for your question. 
gmaster includes Semantic Merge, therefore, if you have Semantic Merge already and wish to transition to gmaster, we will offer a special deal; as pricing for gmaster is being worked out, we cannot be more precise one price points, but existing users will have a fair advantage.

On the contrary, if you are a gmaster user, and hence you work with git, I am not sure why would you need a standalone Semantic Merge license to work with git.  Please, clarify, if I misunderstood.

**Comment by dadhi** [comment]
For instance (real case) I want to compare arbitrary files, not versioned even.

**Comment by dadhi** [comment]
Hi again, elaborating a bit more on using SemanticMerge standalone.

Currently I am using KDiff3 or WinMerge to open file or folders for comparison. Is there such a possibility with GMaster, may be I am missing something?

**Comment by psantosl** [comment]
Yes, semanticmerge can do that. It is like a Kdiff3 on steroids, with semantic powers. Now, what it doesn't do is compare folders... we think you're covered there by the version control (unless you are on svn :P)

**Comment by psantosl** [comment]
And yes, answering to your question back from Oct 3 => you can diff non versioned files too... just open semantic and select the right files, or pass them through the command line.

**Comment by dadhi** [comment]
Cool. Thanks. Now I am happy 👍
